test/uk/gov/hmrc/customs/declarations/stub/controllers/CustomsDataStoreStubControllerSpec.scala

```diff
@@ -28,13 +28,20 @@ class CustomsDataStoreStubControllerSpec extends AnyWordSpec with Matchers {
 
   "GET emailIfVerified" should {
 
-    "return a 200(OK) payload with the expected payload for an EORI number not ending in '99'" in {
+    "return a 200(OK) payload with the expected payload for an EORI number not ending in '99' or '98'" in {
```

Comment:
"or '98'"?? 98 is not covered in this test (but the one you added below), no?

Reply:
As in, return a verified email payload for EORIs ending 00-97, because EORIs ending in 99 or 98 respond with a unverified and undeclared email respectively.